stable/meteor/templates/job.yaml

```diff
@@ -33,6 +33,8 @@ spec:
             envFrom:
               - configMapRef:
                   name: "{{ include "meteor.name" . }}-variables-configmap"
+              - secretRef:
```

Comment:
Please update `values.yaml` and readme to depict these changes.

Reply:
updated `values.yaml` and also `readme`